conformance/conformance.go

```diff
@@ -124,11 +118,7 @@ func handle(req *pb.ConformanceRequest) *pb.ConformanceResponse {
 	switch req.RequestedOutputFormat {
 	case pb.WireFormat_PROTOBUF:
 		var p []byte
-		if isProto3 {
-			p, err = proto.Marshal(&msg)
-		} else {
-			p, err = proto.Marshal(&msg1)
-		}
+		p, err = proto.Marshal(&msg)
```

Comment:
`p, err :=`

Reply:
done

Reply:
I don't see any changes. Did you add this file to the latest commit?

Reply:
I guess not? Fixed now.